src/app/shared/search-box/query-item/query-items.converter.ts

```diff
@@ -102,12 +107,15 @@ export class QueryItemsConverter {
   private createLinkItems(linkTypeIds: string[]): QueryItem[] {
     return this.data.linkTypes.filter(linkType => linkTypeIds.includes(linkType.id))
       .map(linkType => {
-        linkType.collections = [
-          this.data.collections.find(collection => collection.id === linkType.collectionIds[0]),
-          this.data.collections.find(collection => collection.id === linkType.collectionIds[1])
-        ];
+        const collection1 = this.data.collections.find(collection => collection.id === linkType.collectionIds[0]);
+        const collection2 = this.data.collections.find(collection => collection.id === linkType.collectionIds[1]);
+        if (!collection1 || !collection2) {
+          return null;
```

Comment:
I would expect some kind of exception here (or at least `console.error()` call) instead of returning `null`. Because now we are just hiding the problems in the application.